apps/src/code-studio/progress.js

```diff
@@ -27,8 +27,7 @@ progress.renderStageProgress = function (stageData, progressData, scriptName,
     stages: [stageData]
   }, currentLevelId, saveAnswersBeforeNavigation);
 
-  store.dispatch(mergeProgress(_.mapValues(progressData.levels,
-    level => level.submitted ? SUBMITTED_RESULT : level.result)));
+  store.dispatch(mergeProgress(_.mapValues(progressData.levels, level => level.result)));
```

Comment:
Doesn't this break submitted levels? They used to always end up with a result of SUBMITTED_RESULT, now they end up with level.result.